store/src/java/com/zimbra/cs/service/admin/AdminDocumentHandler.java

```diff
@@ -602,10 +600,8 @@ protected Element proxyIfNecessary(Element request, Map<String, Object> context)
                 Account acct = getAccount(prov, AccountBy.fromString(acctElt.getAttribute(AdminConstants.A_BY)),
                         acctElt.getText(), zsc.getAuthToken());
                 if (acct != null && !Provisioning.onLocalServer(acct, reasons)) {
-                    if (zsc.getHopCount() > 2 || (ZimbraLog.soap.isDebugEnabled())) {
-                        ZimbraLog.soap.info("Proxying request:ProxiedAccountElementPath=%s acctElt=%s reasons:%s",
-                                Joiner.on("/").join(xpath), acctElt.toString(), reasons.getReason());
-                    }
+                    ZimbraLog.soap.info("Proxying request:ProxiedAccountElementPath=%s acctElt=%s reasons:%s",
```

Comment:
To be clear (and giving a concrete example of my statement above), I'd suggest changing this to:
```"Proxying request: ProxiedAccountElementPath=%s acctElt=%s reason: %s"```